src/wechaty.ts

```diff
@@ -152,6 +156,8 @@ export class Wechaty extends EventEmitter implements Sayable {
    */
   private static globalInstance: Wechaty
 
+  private static globalPlugins: WechatyPlugin[] = []
```

Comment:
To make the variable name style to be consistent, I'd like to suggest to rename the `globalPlugins` to `globalPluginList`.